frontend/package.json

```diff
@@ -13,6 +13,7 @@
     "react": "^18.2.0",
     "react-cookie": "^4.1.1",
     "react-dom": "^18.2.0",
+    "react-loader-spinner": "^5.4.5",
```

Comment:
This works though there's also a reactstrap spinner that we might prefer https://reactstrap.github.io/?path=/docs/components-spinner--spinner

Reply:
OH it's because i typed `loader` nvm

Reply:
Fixed in [e8f1695](https://github.com/NickPriv/FryRank/pull/17/commits/e8f1695f2a26e47e72060b6f2caab2542be76086).